httpx/_transports/wsgi.py

```diff
@@ -87,7 +87,7 @@ def request(
             "PATH_INFO": unquote(path.decode("ascii")),
             "QUERY_STRING": query.decode("ascii"),
             "SERVER_NAME": host.decode("ascii"),
-            "SERVER_PORT": str(port),
+            "SERVER_PORT": str(port or 80),
```

Comment:
I believe we need to handle HTTP (80) vs HTTPS (443). So let's properly compute the default port separately higher up this method…?

```python
if port is None:
    port = {b"http": 80, b"https": 443}[scheme]
```